frontend/src/components/shared/Modal/Modal.tsx

```diff
@@ -156,11 +161,6 @@ function Modal(props: ModalProps): ReactElement {
       // Will be removed and implemented as the default behavior in the
       // next major version.
       autoFocus={false}
```

Comment:
nit: I believe you can remove this property as well as the comment above. As far as I can see, this is know the default behaviour in baseweb UI.

Reply:
Thanks 👍 I've removed it 👍